Automattic/woocommerce-payments
# Fix Link errors after update

Fixes #4532 

#### Changes proposed in this Pull Request
After latest updates, Link functionality was broken. This PR fixes:
-  autocomplete billing address
 - state/country dropdown
 - Link functionality with blocks








#### Testing instructions





* Navigate to Settings -> Payment Methods and enable Link payment method.
* create an order, go to checkout page and complete all checkout fields. Select also 1click payment under credit card fields
* finish the order
* create another order, go to checkout page and complete the email
* a pop-up should be shown to type the pin ( 0 0 0 0 0 0 for tests )
* all checkout billing fields should be completed
* the order should be completed without errors
* Install the https://github.com/woocommerce/woocommerce-blocks plugin.
* Create a new page and Checkout block.
* Verify that StripeLink is working correctly in the block.
* **check the above steps with different options of `Woo -> Settings -> Shipping -> Shipping options -> Shipping destination`**

-------------------


**Post merge**
